auth/oidc_test.go

```diff
@@ -1255,7 +1255,6 @@ func TestJWTRolesChannels(t *testing.T) {
 				require.Equal(t, base.SetFromArray(login.expectedChannels), user.Channels().AsSet())
 
 				require.Greater(t, user.JWTLastUpdated(), lastUpdateTime)
-				lastUpdateTime = user.JWTLastUpdated()
```

Comment:
I think this is right, because it is checking that this value got updated - are you suggesting I remove this check, or move `lastUpdated` time lower than `user.SetJWTLastUpdated(time.Now())` and use `AssertTimestampGreaterThan`?

Reply:
Nevermind, this doesn't get updated.